src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h

```diff
@@ -886,6 +893,15 @@ namespace sirf {
                 *iter++ = (*iter_x++) / (*iter_y++);
         }
 
+		virtual size_t address() const {
+                    auto *pd_ptr = dynamic_cast<const stir::ProjDataInMemory*>(data().get());
+		    return reinterpret_cast<size_t>(pd_ptr->get_const_data_ptr());
+		}
+		virtual size_t address() {
+                    auto *pd_ptr = dynamic_cast<stir::ProjDataInMemory*>(data().get());
+		    return reinterpret_cast<size_t>(pd_ptr->get_data_ptr());
+		}
```

Comment:
can't see why we need a non-`const` version.

Reply:
removed

Reply:
not here either

Reply:
just removed it